### PR TITLE
Update docs for IBM MQ.

### PR DESCRIPTION
**Pull request by philippkahr**
## What does this PR do?
In the old metricbeat documentation we explicitly list that we only support containerized runtimes for the metricset collection and the reason. [Metricbeat](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-ibmmq.html#_compatibility_23)

I've updated the docs accordingly.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

**Comment by lalit-satapathy** [comment]
@harnish-elastic, Please review the changes!

File: packages/ibmmq/_dev/build/docs/README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ The IBM MQ Integration is used to fetch observability data from [IBM MQ web endp
 
 ## Compatibility
 
-This integration has been tested against `IBM MQ v9.1` and `IBM MQ v9.2`.
+This integration has been tested against `IBM MQ v9.1` and `IBM MQ v9.2`. The ibmmq `qmgr` metricset is compatible with a containerized distribution of IBM MQ (since version 9.1.0).
```
Comment (harnish-elastic) [inline packages/ibmmq/_dev/build/docs/README.md:7]: minor nit: `metricset` -> `data stream`

**Review by harnish-elastic — CHANGES_REQUESTED** [review]

File: packages/ibmmq/_dev/build/docs/README.md (hunk 1 above)
Comment (harnish-elastic) [reply packages/ibmmq/_dev/build/docs/README.md:7]: @philippkahr Could you please update `packages/ibmmq/_dev/build/docs/README.md` as well and perform the `elastic-package check` command from `integrations/packages/ibmmq`

**Review by harnish-elastic — APPROVED** [review]
LGTM

**Comment by lalit-satapathy** [comment]
@philippkahr, Please add a changelog and let's know once the CI is green. @harnish-elastic Please help as needed.

**Comment by harnish-elastic** [comment]
@lalit-satapathy FYI, CI is green now.

**Pull request merged**